### ISSUE TITLE
"case default" syntax error is not caught

### ISSUE THREAD

**Issue by jayfresh** [douglascrockford/JSLint#26]
```
switch(myVar) {
    case 'orange':
        // do something
        break;
    case default:
        // do something else
        break;
}
```

JSlint doesn't pick up the "case default" syntax error.
